match.js

```diff
@@ -53,4 +57,13 @@ class Match {
   players() {
     return this.players_;
   }
+  
+  gameStateAtTime(time){
+    let latestGameState_ = this.gameStateEvents_[0];
+    for (let i = 0; i < this.gameStateEvents_.length; i++) {
+      if (this.gameStateEvents_[i].timestamp >= time) break;
```

Comment:
`> time`, not  `>= time`, we want that state returned if it matches the time exactly.